sources/core/Stride.Core.CompilerServices.Tests/Stride.Core.CompilerServices.Tests.csproj

```diff
@@ -22,7 +22,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\tests\xunit.runner.stride\xunit.runner.stride.csproj" />
-    <ProjectReference Include="..\Stride.Core\Stride.Core.csproj" ReferenceOutputAssembly="false" OutputItemType="Content" CopyToOutputDirectory="Always" />
+    <ProjectReference Include="..\Stride.Core\Stride.Core.csproj" ReferenceOutputAssembly="true" OutputItemType="Content" CopyToOutputDirectory="Always" />
```

Comment:
i had problems here as it searched in shared folder of c# this works too and doesnt make problems with copying stride.core to the output

Reply:
Ideally we don't want the test project to reference Stride.Core to ensure we don't accidentally introduce subtle bugs in case it affected the analyzer run. But maybe it doesn't matter. If referencing it helps always copy the Stride.Core.dll to output to be available to the test compilation, then I think it's fine.